tests/EpisodeManagementServiceTests/CreateUpdateEpisodeTests/CreateUpdateEpisodeTests.cs

```diff
@@ -58,18 +86,18 @@ public async Task Run_Return_InternalServerError_When_Create_Episode_Fails()
         var json = JsonSerializer.Serialize(episode);
         _mockRequest = _setupRequest.Setup(json);
 
-        _mockHttpRequestService.Setup(x => x.SendPost("CreateEpisodeUrl", It.IsAny<string>())).Throws<Exception>();
+        _mockHttpRequestService.Setup(x => x.SendGet(It.IsAny<string>())).Throws<Exception>();
 
         // Act
         var result = await _function.Run(_mockRequest.Object);
 
         // Assert
-        _mockHttpRequestService.Verify(x => x.SendPost("CreateEpisodeUrl", It.IsAny<string>()), Times.Once);
+        _mockHttpRequestService.Verify(x => x.SendGet(It.IsAny<string>()), Times.Once);
```

Comment:
This test should assert no call is made to SendPost

Reply:
Added assertion.